src/pages/library/datasetBuilder/dataset-builder-types.ts

```diff
@@ -63,7 +63,9 @@ export const newCohort = (name: string): Cohort => {
   };
 };
 
-export type ConceptSet = DatasetBuilderType;
+export type ConceptSet = DatasetBuilderType & {
+  featureValueGroupName: string;
+};
```

Comment:
When should we use `&` like this and when should we use `extends`?
```suggestion
export interface ConceptSet extends DatasetBuilderType {
  featureValueGroupName: string;
};
```

It's not clear to me from reading the [typescript docs](https://www.typescriptlang.org/docs/handbook/2/objects.html#interfaces-vs-intersections) when to use one or the other.

Reply:
I am also not sure, I think `extends` feels more correct here though. Have changed it!